Dockerfile

```
@@ -18,6 +18,7 @@ FROM python:2.7
 
 RUN apt-get update && apt-get install libgtk2.0 -y
 RUN pip install opencv-python
+RUN pip install futures
```

We should combine lines 20 and 21 to reduce the number of layers
```suggestion
RUN pip install opencv-python futures
```